docs/cpp/lldb-mi.md

```diff
@@ -54,13 +54,15 @@ If you wish to permanently dismiss this prompt, you can run the following comman
 sudo /usr/sbin/DevToolsSecurity --enable
 ```
 
-## Non-default LLDB.framework paths or custom lldb-mi
-If you wish to use a LLDB.framework that is not installed the way described above, you can copy the `lldb-mi` executable in `~/.vscode/extensions/ms.vscode-cpptools-<version>/debugAdapters/lldb/bin` right next to where the `LLDB.framework` is.
+## Using a LLDB.Framework not installed via XCode
+If you wish to use a LLDB.framework that is not installed the way described above, you can copy the `lldb-mi` executable in `~/.vscode/extensions/ms.vscode-cpptools-<version>/debugAdapters/lldb-mi/bin` right next to where the `LLDB.framework` is.
```

Comment:
instead of "the way described above" i would just say "with XCode" in case you reorder the "above" and forget about this sentence.

Reply:
`right next to where the `LLDB.framework` folder is located.` right? you want it next to the framework folder?